store/tikv/scan.go

```diff
@@ -27,13 +27,14 @@ type Scanner struct {
 	snapshot     *tikvSnapshot
 	batchSize    int
 	valid        bool
+	keyOnly      bool
```

Comment:
Why `keyOnly` is add to `Scanner`?  it's accessible from `Scanner.snapshot.keyOnly` already.

Reply:
It can be omitted, thanks.